homeassistant/components/binary_sensor/device_trigger.py

```diff
@@ -72,6 +72,7 @@
 
 TURNED_ON = [
```

Comment:
Could we maybe generate these instead to avoid this happening again?

```py
TURNED_ON = [trigger[0] for trigger in ENTITY_TRIGGERS.values()]
TURNED_OFF= [trigger[1] for trigger in ENTITY_TRIGGERS.values()]
```

Reply:
It would be a good idea, I'm not that proficient in python yet.

Reply:
Should I update the PR?

Reply:
Yes, please 👍

Reply:
This would work after running the audit
```
TURNED_ON = [trigger[0]["type"] for trigger in ENTITY_TRIGGERS.values()]
TURNED_OFF = [trigger[1]["type"] for trigger in ENTITY_TRIGGERS.values()]
```